tests/compile.sh

```diff
@@ -172,11 +172,11 @@ export CMAKE_FLAGS
 bash -x ${PATHTR}/build.sh
 
 mv ${BUILD_DIR}/ufs_model ${PATHTR}/tests/${BUILD_NAME}.exe
-if [[ "${MAKE_OPT}" == *"DEBUG=Y"* ]]; then
-  cp ${PATHTR}/modulefiles/ufs_${MACHINE_ID}_debug ${PATHTR}/tests/modules.${BUILD_NAME}
-else
-  cp ${PATHTR}/modulefiles/ufs_${MACHINE_ID}       ${PATHTR}/tests/modules.${BUILD_NAME}
-fi
+
+echo "#%Module" > ${PATHTR}/tests/modules.${BUILD_NAME}
+for m in ${modulefile} ; do
+  tail -n +2 ${PATHTR}/modulefiles/${m} >> ${PATHTR}/tests/modules.${BUILD_NAME}
+done
```

Comment:
This hack should not be needed once gocart modules are moved to `ufs_common`.

Reply:
Removed by a244bf9e8588f42982f6cc3b066df7905d2461d5